plugins/by-name/yaml-schema-detect/default.nix

```diff
@@ -0,0 +1,25 @@
+{
+  lib,
+  config,
+  options,
+  ...
+}:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "yaml-schema-detect";
+  packPathName = "yaml-schema-detect.nvim";
+  package = "yaml-schema-detect-nvim";
+
+  description = ''
+    A Neovim plugin that automatically detects and applies YAML schemas for your YAML files using yaml-language-server (yamlls)
+  '';
+
+  settingsExample = {
+    keymap = {
+      refresh = "<leader>yr";
+      cleanup = "<leader>yc";
+      info = "<leader>yi";
+    };
+  };
+
+  maintainers = [ lib.maintainers.FKouhai ];
```

Comment:
Don't know if we really have a rule on this, but it's easier to find above the description.